src/k8s/pkg/k8sd/database/sql/queries/cluster-configs/upsert-config.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO cluster_configs (key, value)
+VALUES (?, ?)
+ON CONFLICT (key) DO UPDATE SET value = EXCLUDED.value;
```

Comment:
Are we sure that this works properly on sqlite? I would recommend adding some test that verifies that the query works as we expect and that we don't end up having conflicting configs in the table.

Reply:
I have a basic unittest that checks if keys can properly be written, read and overwritten. Seems to work, however I'm by far no expert on SQL so if there is any extra test case that I should check, I happily do it.